IM_calculation/Advanced_IM/tests/test_runpy.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#out_dir=/nesi/nobackup/nesi00213/RunFolder/ykh22/test_adv_im/v19p5p8_no_gr_SAC_3and9/Runs/3792018/3792018/test_station/tmp/test_ATC12
```

Comment:
Remove commented lines? (and at end of file?)

Reply:
done